changelog.d/pa-2558.fixed

```diff
@@ -0,0 +1,2 @@
+Go: Fixed a bug where function arguments in a group that share the same type, after the first, would parse as
```

Comment:
I have trouble to understand this. Would be accurate to say that the bug prevented `$ARG1, $ARG2, $ARG3 int` from matching a parameter list like `x, y, z int` ?

Reply:
amended